Dataset Nijmegen

gerritsmedia opened the pull request:
Dear Alejandra,

Here is the dataset generated in Nijmegen. It contains all the metadata. We still need to work on the Celmigration repository, such that we can comment with a link to the raw and analysis datafiles.

With best regards,
Gert-Jan

gerritsmedia committed:
Add files via upload

These files consist of all the metadata of 3D invasion study with HT1080 fibrosarcoma cells. The study consists of three biological replicates of a spheroid invasion assay. The cells are cultured under two different collagen concentrations. Under these conditions, the spheroid assay displays different migration phenotypes depending on the collagen concentration. After cell culture, samples were fixed, fluorescently immunolabeled and imaged by laser scanning confocal microscopy. Datasets are not uploaded to this repository, but can be found using the folling link: .... .

gerritsmedia committed:
Merge pull request #1 from gerritsmedia/gerritsmedia-patch-1

Add files via upload

proccaserra commented on CMSOdataset2018-3DsphrdInvCollHT1080-Bakker/isa/a_2018_3dsphinvasion_ht1080_collagen_spheroid_invasion_assay_microscopy_imaging-N2.txt:
```diff
@@ -0,0 +1,9 @@
+Sample Name	Protocol REF	Parameter Value[imaging modality]	Term Source REF	Term Accession Number	Parameter Value[imaging sequence type]	Term Source REF	Term Accession Number	Parameter Value[observation period]	Term Source REF	Term Accession Number	Unit	Term Source REF	Term Accession Number	Parameter Value[channel definition]	Parameter Value[objective lens]	Parameter Value[numerical aperture]	Raw  ...
```
@gerritsmedia It seems that this ISA Assay File is missing an "Assay Name" (and values) right before the "Raw Data File" Field. The Field is required to pass ISA syntactic validation.

gerritsmedia replied:
I checked it with the Isacreator and indeed it seems that I deleted this particular column. I thought, the assay name is already in the filename itself, why repeat it? But, I can add it again.

agbeltran replied:
That column refers to the specific assay execution/application, while the assay filename is the name of the assay table, containing the details of all the specific assay application. Could you please to add the column back in for this PR?

proccaserra replied:
Some "Assay Name" may have multiple output files hence the field. in your case, it seems indeed somehow duplicating information but it is necessary for it to be present.

agbeltran replied:
Thanks @gerritsmedia - feel free to update your repository and the pull request. There are also issues with the protocols and parameter names (e.g. (micro)environment). The ```Protocol REF``` column refers to the protocols defined in the investigation file and the parameters have to match. To link to the documents describing the protocols, it would be better to use the ```Study Protocol URI``` if possible.

agbeltran replied:
Some of the syntactic issues: the protocol "Appendix I" was missing the following parameters
"cell culture CO2 partial pressure;cell culture temperature;cell culture antibiotics;cell culture serum concentration;cell culture serum;cell culture medium;cell culture vessel;(micro)environment"

The protocol "Appendix II Microscopy" was defined as "Appendix II Microscopy.pdf" in the investigation file and they need to match in investigation and study files.

gerritsmedia replied:
I will update the files and change the protocol names to *.pdf to make it consistent over the dataset. For the rest, the parameters mentioned above (for appendix I) are all included, however, not with the exact terms. Do I need to include these terms literally? In that case, I need to make some slight adaptations to the protocol in the way it is written. 

What do you mean with (micro)environment? I could adapt the section 'Spheroid embedding in 3D collagen', such that it includes the term '(micro)environment' literally. Could I freely specify the (micro)environment as 'reconstituted fibrillar rattail collagen', or is there a standard term to specify the (micro)environment?

gerritsmedia replied:
Hi, I updated the files, except for the protocol, I will work on that later. Are the isa tab files correct now?

gerritsmedia replied:
The file Appendix I 3D spheroid culture.pdf is also updated now.

agbeltran commented:
Thanks @gerritsmedia  @proccaserra - I'm checking the files and adding automated validation.

gerritsmedia committed:
Add files via upload

Updated version, with the assay name column included and modification to the protocol names to make them match among the different isa tab files.

gerritsmedia committed:
Add files via upload

This is the updated version of the protocol with the missing cell culture mesh terms included. Also, the term (micro)environment is included.
